Source/WebCore/css/counterStyles.css

```diff
@@ -443,6 +441,57 @@
     /* 'ㄱ' 'ㄴ' 'ㄷ' 'ㄹ' 'ㅁ' 'ㅂ' 'ㅅ' 'ㅇ' 'ㅈ' 'ㅊ' 'ㅋ' 'ㅌ' 'ㅍ' 'ㅎ' */
 }
 
+/** Ethiopic ready-made Counter Styles **/
+
+/* FIXME: The W3C definition adds '\12F8' ('ዸ') between '\12F0' ('ደ') & '\1308' ('ገ') */
+@counter-style afar {
+    system: alphabetic;
+    symbols: '\1200' '\1208' '\1210' '\1218' '\1228' '\1230' '\1260' '\1270' '\1290' '\12A0' '\12A8' '\12C8' '\12D0' '\12E8' '\12F0' '\1308' '\1338' '\1348';
+    /* 'ሀ' 'ለ' 'ሐ' 'መ' 'ረ' 'ሰ' 'በ' 'ተ' 'ነ' 'አ' 'ከ' 'ወ' 'ዐ' 'የ' 'ደ' 'ገ' 'ጸ' 'ፈ' */
+    suffix: '\1366 '; /* ፦ */
+}
+
+/* FIXME: The W3C definition calls it "haleme", check if it is a typo */
```

Comment:
I'm not sure why https://w3c.github.io/predefined-counter-styles/#ethiopic-haleme calls it `Haleme`, but I could find a number of references on internet using the word `Halehame`, also referred as `Debtera` sometimes. And this is also the name used for the `list-style-type` itself. So I guess we can remove this `FIXME`. I couldn't find much with `Haleme`, so it might be good to create an spec issue, although this is in the informative part.

http://yacob.org/papers/DanielYacob-IUC19.pdf
https://www.senamirmir.org/interviews/theme/5-2001/gh/ws.html